docs/src/content/docs/fr/reference/frontmatter.md

```diff
@@ -111,20 +111,44 @@ hero:
 ---
 ```
 
+Vous pouvez afficher différentes versions de l'image de premier plan en mode clair et sombre.
+
+```md
+---
+hero:
+  image:
+    alt: Un logo scintillant aux couleurs vives
+    dark: ../../assets/logo-dark.png
+    light: ../../assets/logo-light.png
+---
+```
+
 #### `HeroConfig`
 
 ```ts
 interface HeroConfig {
   title?: string;
   tagline?: string;
-  image?: {
-    alt?: string;
-    // Chemin relatif vers une image dans votre dépôt.
-    file?: string;
-    // HTML brut à utiliser dans l'emplacement de l'image.
-    // Il peut s'agir d'une balise `<img>` personnalisée ou d'une balise `<svg>` en ligne.
-    html?: string;
-  };
+  image?:
+    | {
+        // Chemin relatif vers une image dans votre dépôt.
+        file: string;
+        // Alternative textuelle pour rendre l'image accessible aux technologies d'assistance.
```

Comment:
Reference for "Alternative textuelle": https://fr.wikipedia.org/wiki/Alternative_textuelle